calico/getting-started/kubernetes/windows-calico/quickstart.mdx

```diff
@@ -465,7 +475,8 @@ c:\PrepareNode.ps1 -KubernetesVersion v1.26.6
 kubeadm token create --print-join-command
 ```
 
-- Edit the join command by appending `--cri-socket "npipe:////./pipe/containerd-containerd"` and update the kubeadm.exe path to `c:\k\kubeadm.exe`.
+- Edit the join command by updating the kubeadm.exe path to `c:\k\kubeadm.exe`.
+- If using a kubernetes version below v1.25, also append `--cri-socket "npipe:////./pipe/containerd-containerd"` to the join command.
   An example join command:
```

Comment:
At the end of the Requirements section, you have the bullet about "The Windows nodes have joined the cluster".
It would be easier to follow if you moved the two items about joining the cluster right after that bullet. And push the one about "To install containerd on the Windows node and configure the containerd service" after.

Reply:
Hm, I don't think that would work. I'm not sure if that's clear enough (appreciate suggestions to do so), but all of these steps are part of (or prereqs to) joining the cluster...